assets/queries/k8s/drop_capabilities_psp/metadata.json

```diff
@@ -0,0 +1,8 @@
+{
+  "id": "Drop_Capabilities_Psp",
+  "queryName": "Containers dont have NET_RAW or ALL as Drop Capabilities",
```

Comment:
folder, id and query name should be the same. A more suitable query name would be: NET_RAW Capabilities Enabled